src/mito2/src/region.rs

```diff
@@ -526,6 +522,18 @@ impl OpeningRegions {
 
 pub(crate) type OpeningRegionsRef = Arc<OpeningRegions>;
 
+/// Single region stats.
+#[derive(Default, Debug, Clone)]
+pub(crate) struct Stats {
```

Comment:
Maybe we can rename it to ManifestStats?

Reply:
+1